csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css

```diff
@@ -0,0 +1,132 @@
+
+
+body{

```

Comment:
I think I would prefer it if you put these 2 rules that have relative paths in the html itself and use the `static` django template filter to link them.

It is generally best practice to put the css rules in the css file, I make an exception for rule that use paths to other files, just cause it makes it easier to debug in pycharm.

Reply:
@JohnWu966 and you still need to deal with this I believe.

Reply:
As far as I've know, its not possible. Linking an image using `static` requires two sets of quotation marks. Setting images on the background of a div is possible in the html file itself if you use in-line CSS, but that requires the use of an additional set of quotation marks. Unless there's a third type of quotation mark that I'm unaware of, then its not possible to use static images and in-line css in conjunction.

Reply:
it is possible

https://github.com/CSSS/csss-site/blob/master/csss-site/src/csss/templates/csss/header.html#L18-L28

Reply:
you still need to remove the rules from the CSS